### DISCUSSION TITLE
`install.sh` fails when `$HOME/.cargo/bin` does not exist

### DISCUSSION THREAD

**Discussion by bengsparks** [SeaQL/FireDBG.for.Rust#13] [Bug]
`install.sh` does not check for the existence of the [`$HOME/.cargo/bin`](https://github.com/SeaQL/FireDBG.for.Rust/blob/f30ff87a985ccf08439d1c13b2f51d1bcfee5972/install.sh#L50) prior to [`mv`ing](https://github.com/SeaQL/FireDBG.for.Rust/blob/f30ff87a985ccf08439d1c13b2f51d1bcfee5972/install.sh#L71) the contents of the folder generated by `mktemp`.

(anonymised) Output:
```bash
$ curl https://raw.githubusercontent.com/SeaQL/FireDBG.for.Rust/main/install.sh -sSf | sh
info: downloading FireDBG from `https://github.com/SeaQL/FireDBG.for.Rust/releases/download/1.74.1/aarch64-darwin.tar.gz`
info: installing FireDBG binaries to `$HOME/.cargo/bin`
mv: rename /var/folders/mj/_03p5tkj30g1y7518fx_jydm0000gn/T/tmp.C61ozBF0pI/firedbg-lib to $HOME/.cargo/bin/firedbg-lib: No such file or directory
FireDBG: command failed: mv /var/folders/mj/_03p5tkj30g1y7518fx_jydm0000gn/T/tmp.C61ozBF0pI/firedbg-lib $HOME/.cargo/bin/firedbg-lib
```

A likely fix would be to ensure its existence with [mkdir -p](https://github.com/SeaQL/FireDBG.for.Rust/blob/f30ff87a985ccf08439d1c13b2f51d1bcfee5972/install.sh#L55C5-L55C28) or similar?

**Comment by billy1624** [comment]
Hey @bengsparks, thanks for the suggestions!! I just edited the install script https://github.com/SeaQL/FireDBG.for.Rust/commit/19b5a04c3641a16da79e60155412ed151e748b62
Could you please try again?

**Reply by bengsparks** [reply]
Yes that seems to have resolved it. Thank you!
